addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -305,6 +350,76 @@ class AwsCfnService extends Service {
     }
   }
 
+  async checkPendingAccounts(requestContext) {
```

Comment:
It looks like this method actually onboard pending accounts? Should it be renamed to that? `onboardPendingAccounts`.

`checkPendingAccounts` would lead me to think it's a method that doesn't mutate state, but this method actually does change state.

Reply:
that's true, I can change this--it calls the attemptOnboard function to actually do the onboarding, but I agree since it's calling that as a subfunction the parent function is then mutating state